app/views/admin/ofsted/_shortcut-nav.html.erb

```diff
@@ -1,31 +1,34 @@
 <nav class="shortcuts">
-    
-    <%= render partial: "shared/shortcuts_item", 
-    locals: {
-        state: current_page?(controller: 'admin/ofsted', action: 'index') && !params[:archived].present?, 
-        link: admin_ofsted_index_path,
-        count: @ofsted_counts.dig(:all, :feed),
-        label: "All"
-    } %>
-<span class="shortcuts__divider">|</span>
-    
+
     <%= render partial: "shared/shortcuts_item", 
-    locals: {
-        state: current_page?(controller: 'admin/ofsted', action: 'pending') , 
-        link: pending_admin_ofsted_index_path,
-        count: @ofsted_counts.dig(:all, :pending),
-        label: "Pending"
-    } %>
+        locals: {
+            state: current_page?({controller: 'admin/ofsted', action: 'index'}) && !params[:archived].present?,
```

Comment:
This diff is really ugly because I fixed some indentation while I was here, but the main change is the added curly braces in the `current_page?` method calls in prep for [Ruby 3's positional and keyword arg changes](https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/)